### PR TITLE
A button for indicating when questions associated with a ticket have been answered and are done

### PR DESCRIPTION
**Pull request by fwald**
When clicking the done button the ticket will automatically be set to completed. 

Also did some minor changes to how we handle fetching tickets. Now by default we only fetch unresolved tickets. A user has to specifically request the resolved tickets in order to see them.

### CODE REVIEW

File: src/components/tickets/ticket/ticketBody.tsx [hunk 1]
```diff
@@ -197,6 +199,10 @@ export class TicketBody extends React.Component<ITicketBody, ITicketBodyState> {
                 this.props.type,
               )
             : null}
+
+            //TODO: INSERT NEW COMPONENT HERE !!!
```
Comment (vonElfvin) [inline src/components/tickets/ticket/ticketBody.tsx]: ?

File: src/components/tickets/tickets.tsx [hunk 2]
```diff
@@ -31,7 +32,11 @@ export class Tickets extends React.Component<ITickets, {}> {
             return null
           }}
         </Query>
-        <Query<any> query={GET_TICKETS} pollInterval={5000}>
+        <Query<any> 
+          query={GET_TICKETS} 
+          variables={{onlyResolvedTickets: (this.props.filter.status == TicketStatus.RESOLVED)}}
```
Comment (vonElfvin) [inline src/components/tickets/tickets.tsx]:
```suggestion
          variables={{status: TicketStatus.RESOLVED}} 
```

File: src/features/taskmanager/queries.ts [hunk 3]
```diff
@@ -13,8 +13,8 @@ export const CREATE_TICKET = gql`
 `
 
 export const GET_TICKETS = gql`
-  query GetTickets {
-    tickets {
+  query GetTickets ($onlyResolvedTickets: Boolean ){
```
Comment (vonElfvin) [inline src/features/taskmanager/queries.ts]:
```suggestion
  query GetTickets ($ticketStatus: TicketStatus ){
```

File: src/components/tickets/ticket/ticketBody.tsx (hunk 1 above)
Comment (fwald) [reply src/components/tickets/ticket/ticketBody.tsx]: oops

**Pull request merged**